ets-ecole-de-technologie-superieure.csl

```diff
@@ -88,23 +81,21 @@
         </group>
       </if>
       <else>
-        <text variable="container-title" form="short" font-style="italic" suffix=", "/>
+        <text variable="container-title" form="short" font-style="italic"/>
       </else>
     </choose>
   </macro>
   <macro name="title">
-    <group delimiter=" ">
-      <choose>
-        <if type="chapter paper-conference article-journal article-magazine article-newspaper webpage thesis" match="none">
-          <text variable="title" font-style="italic"/>
-        </if>
-        <else>
-          <group>
-            <text variable="title" quotes="true"/>
-          </group>
-        </else>
-      </choose>
-    </group>
+    <choose>
+      <if type="chapter paper-conference article-journal article-magazine article-newspaper webpage post post-weblog thesis" match="none">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <group>
+          <text variable="title" prefix="« " suffix=" »"/>
```

Comment:
Yes. This is not handled correctly with our style because when we put quotes="true", the punctuation mark appears inside the double quotes which is incorrect in our style